src/__tests__/screens/authScreen.test.tsx

```diff
@@ -22,14 +22,22 @@ describe('Auth Screen Test Suite', async () => {
         render(<RouterProvider router={router} />);
 
         await waitFor(() => screen.getByTestId('featured-search-heading'));
+        // open user menu
+        await user.click(screen.getByTestId('menu-button'));
+        // wait for the menu to be displayed
+        await waitFor(() => screen.getByTestId('menu-appbar'));
         // navigate to login
         await user.click(screen.getByText('Login'));
         await waitFor(() => screen.getByTestId('login-heading'));
         expect(screen.getByText('Password')).toBeInTheDocument();
         expect(screen.getByText('Email')).toBeInTheDocument();
         // go back to homepage
-        await user.click(screen.getByText('Home'));
+        await user.click(screen.getByTestId('home'));
```

Comment:
nit:
If we can avoid using testIds, we should. If not, the ID should explain the element a bit more. In the case, I think `streamability-nav-logo` might make more sense. 
```suggestion
        await user.click(screen.getByText('Streamability'));
```

Reply:
Will do.